wof/wofpy_config.py

```diff
@@ -16,62 +16,82 @@
 
 Usage:
     wofpy_config INDIR
-    wofpy_config INDIR [--mode=<development,production>]
+    wofpy_config INDIR [--mode=<development,production>] [--overwrite=<soft,hard,none>]
```

Comment:
Maybe we could make `soft` the default and drop the extra complexity of `none`. Experienced user would not expect to get things added into an existing directory, but if we write that in the docs it should be OK b/c there won't be loos of data.